zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectsService.kt

```diff
@@ -0,0 +1,194 @@
+package com.ritense.objectmanagement.service
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.ritense.objectenapi.ObjectenApiPlugin
+import com.ritense.objectenapi.client.ObjectRecord
+import com.ritense.objectenapi.client.ObjectRequest
+import com.ritense.objectenapi.client.ObjectWrapper
+import com.ritense.objectenapi.client.ObjectsList
+import com.ritense.objectmanagement.domain.ObjectManagement
+import com.ritense.objectmanagement.repository.ObjectManagementRepository
+import com.ritense.objecttypenapi.ObjecttypenApiPlugin
+import com.ritense.plugin.service.PluginService
+import mu.KotlinLogging
+import org.springframework.data.domain.PageRequest
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import java.net.URI
+import java.time.Instant
+import java.time.LocalDate
+import java.util.UUID
+
+@Service
+class ObjectsService(
```

Comment:
Can you define this as a bean in the autoconfiguration?

Reply:
Also, this class could be called `ObjectsFacade`

Reply:
@marijnritense I have processed your feedback. I have also removed the caching, removing code that became unnecessary as a result, and refactored some code to reduce the number of calls to the repositories.